utils/create-empty-form-data.ts

```diff
@@ -116,7 +116,7 @@ export const createEmptyContactField = {
   phones: (): PhoneUpdate => ({
     key: uuid(),
     id: 0,
-    country_code: 1,
+    country_code: null,
```

Comment:
Would it be better to set this to `us` instead of `null`? Does React throw any controlled input errors with this?